topic-operator/src/main/java/io/strimzi/operator/topic/ReplicasChangeHandler.java

```diff
@@ -122,8 +132,9 @@ public List<ReconcilableTopic> requestOngoingChanges(List<ReconcilableTopic> rec
 
         try {
             LOGGER.debugOp("Sending user tasks request, Tasks {}", groupByUserTaskId.keySet());
+            Timer.Sample timerSample = TopicOperatorUtil.startExternalRequestTimer(metrics, config.enableAdditionalMetrics());
             UserTasksResponse utr = cruiseControlClient.userTasks(groupByUserTaskId.keySet());
-
+            TopicOperatorUtil.stopExternalRequestTimer(timerSample, metrics::cruiseControlUserTasks, config.enableAdditionalMetrics(), config.namespace());
```

Comment:
I'd guess you should also stop the time if an exception is thrown?

Reply:
Right. Thanks for catching this.